lib/rubocop/cop/minitest/focus.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Enforces tests are not focused
```

Comment:
Please add a period at the end of the sentence.

```suggestion
      # Enforces tests are not focused.
```